HVZ.Persistence/Models/Organization.cs

```diff
@@ -49,7 +48,12 @@ public class Organization : IdEquatable<Organization>
     /// </summary>
     public Instant CreatedAt { get; init; }
 
-    public Organization(string id, string name, string ownerid, HashSet<string> moderators, HashSet<string> administrators, HashSet<Game> games, string? activegameid, Instant createdat, string url)
+    /// <summary>
+    ///  Additional information about an org.
+    /// </summary>
+    public string Description { get; set; }
+
+    public Organization(string id, string name, string ownerid, HashSet<string> moderators, HashSet<string> administrators, HashSet<Game> games, string? activegameid, Instant createdat, string url, string description = "")
```

Comment:
change the argument to `string? description = null` then in the assignment below, check for null and provide a generic org description. I am using null specifically so that it is possible to set the description to an empty string if that is wanted